FastSimulation/SimplifiedGeometryPropagator/interface/FastTrackDeDxProducer.h

```diff
@@ -0,0 +1,98 @@
+#ifndef TrackRecoDeDx_FastTrackDeDxProducer_H
```

Comment:
Please use an include guard matching the full file name subsystem_package_fileName_h

Reply:
added FASTSIM_ to include guard